### PR TITLE
Add Catboy API to Anime Category

### PR DESCRIPTION
**Pull request by Tech1k**
Add Catboy API to Anime Category. This API provides images of anime nekos and other neat things like gifs.

- [x] My submission is formatted according to the guidelines in the [contributing guide](/CONTRIBUTING.md)
- [x] My addition is ordered alphabetically
- [x] My submission has a useful description
- [x] The description does not have more than 100 characters
- [x] The description does not end with punctuation
- [x] Each table column is padded with one space on either side
- [x] I have searched the repository for any relevant issues or pull requests
- [x] Any category I am creating has the minimum requirement of 3 items
- [x] All changes have been [squashed][squash-link] into a single commit

[squash-link]: <https://github.com/todotxt/todo.txt-android/wiki/Squash-All-Commits-Related-to-a-Single-Issue-into-a-Single-Commit>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -147,6 +147,7 @@ API | Description | Auth | HTTPS | CORS |
 | [AnimeChan](https://github.com/RocktimSaikia/anime-chan) | Anime quotes (over 10k+) | No | Yes | No |
 | [AnimeFacts](https://chandan-02.github.io/anime-facts-rest-api/) | Anime Facts (over 100+) | No | Yes | Yes |
 | [AnimeNewsNetwork](https://www.animenewsnetwork.com/encyclopedia/api.php) | Anime industry news | No | Yes | Yes |
+| [Catboy API](https://catboys.com/api) | Neko images, funny GIFs & more | No | Yes | Yes |
```
Comment (pawelborkar) [inline README.md:150]:
```suggestion
| [Catboy](https://catboys.com/api) | Neko images, funny GIFs & more | No | Yes | Yes |
```

Comment (pawelborkar) [reply README.md:150]: Title should not end with "... API"

**Review by pawelborkar — CHANGES_REQUESTED** [review]
Please commit the suggestions stated below and don't forget to squash all the commits into a single one

File: README.md (hunk 1 above)
Comment (Tech1k) [reply README.md]: @pawelborkar updated as requested!

**Review by pawelborkar — APPROVED** [review]
Thanks for the contribution @Tech1k

**Pull request merged**